sysvabi64/sysvabi64.rst

```diff
@@ -1417,9 +1417,7 @@ GNU C Library IFUNC interface
 The GNU C Library (glibc) ``sys/ifunc.h`` header provides the
 necessary type definitions for defining IFUNC resolvers. The resolver
 is passed at least one, and at most 2 parameters dependent on the
-glibc IFUNC ABI version. A flag is set in the first parameter if a
-second parameter is available. The parameters provide access to the
-HWCAP_ flags.
+glibc IFUNC ABI version.
```

Comment:
probably clearer to move this sentence to the start of the next paragraph, after the code?